src/app/templates/index.html

```diff
@@ -1,75 +1,169 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Bingo Card Maker</title>
-  <link rel="stylesheet" href="{{ url_for('static', filename='styles.css') }}">
+    <meta charset="UTF-8">
+    <title>Bingo Card Generator</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="src/app/static/styles.css">
```

Comment:
loading this css will 404. when using a template you should use 
`<link rel="stylesheet" href="{{ url_for('static', filename='styles.css') }}">` to load static files.

Reply:
this line is also duplicated at the end of the file